src/App.jsx

```diff
@@ -14,33 +16,102 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const goods = goodsFromServer.map((name, id) => ({
+  name,
+  id,
+}));
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const SORT_FIELD_ALPHABETIC = 'Alphabetically';
+const SORT_FIELD_BYLENGTH = 'ByLength';
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+function getPreparedGood(allGoods, { sortField, isReversed }) {
```

Comment:
In this case, you can move this function to a separate file then import and use it inside the component as a helper/util function. ( for future)